yt/data_objects/data_containers.py

```diff
@@ -1513,13 +1513,19 @@ def _ds_hold(self, new_ds):
     def _chunked_read(self, chunk):
         # There are several items that need to be swapped out
         # field_data, size, shape
+        obj_field_data = []
```

Comment:
I'm not 100% convinced this is right.  Can you remind me which object composes `chunk.objs` in the issue we're dealing with?

This also may result in caching all field data from all grid objects in a grid frontend.

Reply:
In the issue they're instances of `OctreeSubset`.